go.mod

```diff
@@ -14,33 +14,33 @@ require (
-	github.com/openstack-k8s-operators/glance-operator/api v0.3.1-0.20240703160047-b10f37be3c50
-	github.com/openstack-k8s-operators/heat-operator/api v0.3.1-0.20240703115602-422a5e8a4337
-	github.com/openstack-k8s-operators/horizon-operator/api v0.3.1-0.20240703115846-c761682ad2ad
-	github.com/openstack-k8s-operators/infra-operator/apis v0.3.1-0.20240701051058-e23cdfd81161
-	github.com/openstack-k8s-operators/ironic-operator/api v0.3.1-0.20240703115847-33328239d321
-	github.com/openstack-k8s-operators/keystone-operator/api v0.3.1-0.20240703053916-a2bf14020ef4
-	github.com/openstack-k8s-operators/lib-common/modules/ansible v0.3.1-0.20240624132705-6c8da3c0bbfd
-	github.com/openstack-k8s-operators/lib-common/modules/certmanager v0.0.0-20240624132705-6c8da3c0bbfd
-	github.com/openstack-k8s-operators/lib-common/modules/common v0.3.1-0.20240624132705-6c8da3c0bbfd
-	github.com/openstack-k8s-operators/lib-common/modules/storage v0.3.1-0.20240624132705-6c8da3c0bbfd
-	github.com/openstack-k8s-operators/lib-common/modules/test v0.3.1-0.20240624132705-6c8da3c0bbfd
-	github.com/openstack-k8s-operators/manila-operator/api v0.3.1-0.20240703123716-e68f23707bf7
-	github.com/openstack-k8s-operators/mariadb-operator/api v0.3.1-0.20240626205513-5c175a95f408
-	github.com/openstack-k8s-operators/neutron-operator/api v0.3.1-0.20240703113929-8201d1454a2f
-	github.com/openstack-k8s-operators/nova-operator/api v0.3.1-0.20240703151430-8f8ca6512933
-	github.com/openstack-k8s-operators/octavia-operator/api v0.3.1-0.20240703143816-b6000f653f1c
-	github.com/openstack-k8s-operators/openstack-ansibleee-operator/api v0.3.1-0.20240627174704-83f16917cf99
-	github.com/openstack-k8s-operators/openstack-baremetal-operator/api v0.3.1-0.20240703054730-75382e776839
+	github.com/openstack-k8s-operators/barbican-operator/api v0.4.1-0.20240705080032-2a515074bcea
+	github.com/openstack-k8s-operators/cinder-operator/api v0.4.1-0.20240708165337-4141145a0e7b
+	github.com/openstack-k8s-operators/designate-operator/api v0.1.1-0.20240704231514-c2b1728e46b7
+	github.com/openstack-k8s-operators/glance-operator/api v0.4.1-0.20240708125058-56e93700255f
+	github.com/openstack-k8s-operators/heat-operator/api v0.4.1-0.20240704231515-a1bb7731fd73
+	github.com/openstack-k8s-operators/horizon-operator/api v0.4.1-0.20240704220914-63a9fd21e69b
+	github.com/openstack-k8s-operators/infra-operator/apis v0.4.1-0.20240704231227-c7ee9d3de988
+	github.com/openstack-k8s-operators/ironic-operator/api v0.4.1-0.20240704220915-746cf7c339c5
+	github.com/openstack-k8s-operators/keystone-operator/api v0.4.1-0.20240704220917-e65444cf5f26
+	github.com/openstack-k8s-operators/lib-common/modules/ansible v0.4.0
+	github.com/openstack-k8s-operators/lib-common/modules/certmanager v0.4.0
+	github.com/openstack-k8s-operators/lib-common/modules/common v0.4.0
+	github.com/openstack-k8s-operators/lib-common/modules/storage v0.4.0
+	github.com/openstack-k8s-operators/lib-common/modules/test v0.4.0
```

Comment:
we might need to manually bump from v0.4.0 to v0.4.1.-<pseudoversion> when a new commit lands, but this remains to be seen.